core/src/session.rs

```diff
@@ -117,7 +105,8 @@ impl Session {
             audio_key: OnceCell::new(),
             channel: OnceCell::new(),
             mercury: OnceCell::new(),
-            handle,
+            dealer: tokio::sync::OnceCell::new(),
```

Comment:
Not sure if it makes sense to store the dealer here.

(BTW tokio's async OnceCell was added after I created a feature request, so I had to use it)